pages/common/jf.md

```diff
@@ -0,0 +1,16 @@
+# jf
+
+> Use JF CLI for interfacing with JFrog products like Artifactory, Xray, Distribution, Pipelines and Mission Control.
```

Comment:
```suggestion
> Interface with JFrog products like Artifactory, Xray, Distribution, Pipelines and Mission Control.
```
How about this? (As we are currently moving from the usage of `CLI` term in page descriptions)

Reply:
oh that's a good point. I like your suggestion